keyboards/misterknife/knife66/knife66.c

```diff
@@ -0,0 +1 @@
+#include "knife66.h"
```

Comment:
Could you add a license header here?

Sample here:

https://github.com/qmk/qmk_firmware/blob/bc010d62cab9a632d6338218e648aa37a8814c1d/quantum/template/base/keymaps/default/keymap.c#L1-L15

Reply:
done